### ISSUE TITLE
Add `aria-selected` to default dropdown selections on the Dashboard

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5126]
## Area or Component
Dashboard

## Quick Summary
The default selected values for the "Testing facility" and "Date range" dropdowns do not have states or properties that are necessary for screen reader users to interact with or understand the content conveyed by the element.

### How to Replicate
- Load the dashboard
- Notice that by default, "All facilities" and "Last week (7 days)" are selected but that state information is not conveyed to screen readers

## Accessibility Violation

[States/Properties: The element has missing or incorrect states or properties](https://cdc-devtools.dequecloud.com/rule-help/en/aria-state-property-missing)

## Action Requested
Add `aria-selected="true"` to the options that are selected by default

## Screenshots (optional)
![Screen Shot 2023-01-25 at 16.31.30.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/167ead3c-3cf7-4d4f-9b21-c7099d9aa173)

## Source Code (optional)

[which file or line is the issue attached to?]: #

## Original Deque Issue
[States/Properties: The element has missing or incorrect states or properties](https://cdc-devtools.dequecloud.com/rule-help/en/aria-state-property-missing)
